hydra-node/src/Hydra/Chain/CardanoClient.hs

```diff
@@ -76,6 +76,7 @@ buildTransaction networkId socket changeAddress utxoToSpend collateral outs = do
   pure $
     second balancedTxBody $
       makeTransactionBodyAutoBalance
+        ShelleyBasedEraBabbage
```

Comment:
Should: use at least `shelleyBasedEra` to not mention `Babbage`

Could: shim `makeTransactionBodyAutoBalance` to not take this witness but produce it with `IsShelleyBasedEra(shelleyBasedEra)`

Reply:
```suggestion
        shelleyBasedEra
```